startup-prod.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+BASH_ENV=.env_app
+mkdir -p /vogon/logs
+python -m pip install -r requirements.txt
+python manage.py migrate
+python manage.py collectstatic --noinput
+exec supervisord -c /etc/supervisor/conf.d/supervisord.conf
```

Comment:
supervisor should be installed as a service so you can start it as such (`systemctl start supervisor` I believe)